weather-map-wk7/README.md

```diff
@@ -0,0 +1,44 @@
+# Weather Forecast app
+
+> This app is a fullstack app build using React Frontend and Express Backend with hardcode data. This project doesn't render real-time data
```

Comment:
This description is no longer accurate, as you are now using an external API and presenting real-time data to the user.